tests/test_math.cpp

```diff
@@ -617,7 +617,7 @@ MainLoop *test() {
 
 	List<String> args;
 	args.push_back("-l");
-	Error err = OS::get_singleton()->execute("/bin/ls", args, true, nullptr, &ret);
+	Error err = OS::get_singleton()->execute("/bin/ls", args, &ret);
```

Comment:
not sure why this is here, but it's fine for now should probably be re-evaluated in a separate PR.

Reply:
That does look like a copy paste error from OS tests to math tests. Could be removed, but this can indeed be done in another PR.